hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -511,12 +492,52 @@
       stage: V2
       tests: []
     }
+    {
+      name: enable
+      desc: '''
+            Verify that DUT connects to the USB when enable is set.
+
+            - set USB_CTRL_enable pin high.
+            - Make sure 'phy_config.pinflip' isn't set.
+            - Verify that the 'usb_dp_pullup_o' pin goes high.
+            '''
+      stage: V2
+      tests: ["usbdev_enable"]
+    }
+    {
+      name: resume_link_active
+      desc: '''
+            Verify that when Write a 1 to this bit to instruct usbdev to jump to the LinkResuming state
+            then write will only have an effect when the device is in the LinkPowered state.
```

Comment:
I'm not completely sure I understand this description. Is this saying that we should verify that setting the `resume_link_active` bit causes usbdev to jump to the `LinkResuming` state? (I think there should probably be a full stop at this point). Then are we saying a write will not have any effect until the device gets to LinkPowered?

Reply:
I'll leave this test description to be addressed in #22129.